src/helpers.rs

```diff
@@ -0,0 +1,61 @@
+#![cfg(test)]
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{to_json_binary, Addr, CosmosMsg, StdResult, WasmMsg};
+
+use crate::msg::ExecuteMsg;
+use crate::msg::SudoMsg;
+
+/// CwTemplateContract is a wrapper around Addr that provides a lot of helpers
+/// for working with this.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct RateLimitingContract(pub Addr);
```

Comment:
since this struct and its methods are only used in integration_tests.rs, I suggest moving these to that file for better structure and understanding that these are for tests only. that would also justify the unwrap() usage below